<launch>
    <env name="ROSCONSOLE_CONFIG_FILE" value="$(find walker_planner)/config/rosconsole.config" />

    <arg name="debug" default="false"/>
    <arg name="rvizconfig" default ="$(find walker_planner)/config/urdf.rviz" />
    <arg name="file_name" default="multi_room_map.env"/>


    <include file="$(find walker_planner)/launch/planning_context_walker.launch"/>

    <arg unless="$(arg debug)" name="launch-prefix" value=""/>
    <arg     if="$(arg debug)" name="launch-prefix" value="gdb -ex run --args"/>
    <env name="SMPL_VISUALIZE_CONFIG_FILE" value="$(find walker_planner)/config/smpl_visual.ini"/>

    <node pkg="walker_planner" type="generate_map" name="generate_map" args="-$(arg file_name)"
            launch-prefix="$(arg launch-prefix)" output="screen" required="true">
        <param name="planning_frame" value="dummy_base"/>
        <rosparam command="load" file="$(find walker_planner)/env/multi_room_map.yaml" />

        <!-- planner params -->
        <rosparam command="load" file="$(find walker_planner)/config/walker_right_arm.yaml"/>
    </node>
</launch>
